### PR TITLE
Add support for lerna monorepos

### PR DESCRIPTION
**Pull request by jaredpalmer**
Add support for lerna monorepos regardless of whether actual npm/yarn workspaces are being used along with it.

### CODE REVIEW

**Comment by jaredpalmer** [comment]
Only nuance to this is that when backfill goes to parse the lockfile when using lerna without yarn/npm workspaces, the root lockfile might not contain the full set of all deps (some may live in the package-level lockfiles). When using lage, this actually doesn't end up mattering though because the package's lockfiles live in git and so their contents hash comes from `git ls-tree` anyways. Regardless, I think this should still be added to workspace tools

**Review by kenotron — COMMENTED** [review]
This is good idea! I'll add it. Not sure about where the lockfile change is coming from tho, @jaredpalmer - can you take a look at that again?

File: yarn.lock [hunk 1]
```diff
@@ -578,6 +578,11 @@
   dependencies:
     "@types/node" "*"
 
+"@types/git-url-parse@^9.0.0":
```
[the remark this answers is not in the document]
Comment (kenotron) [reply yarn.lock:581]: oh this is due to merge.

**Comment by kenotron** [comment]
Merged, 🚀 it!

**Pull request merged**